docs/quickstart.md

```diff
@@ -2,11 +2,11 @@
 
 ## I want to explore the plugin without configuring anything.
 
-[Launch the plugin in WordPress Playground](https://wordpress-playground.atomicsites.blog/?blueprint-url=https://raw.githubusercontent.com/Automattic/remote-data-blocks/trunk/blueprint.json) and explore. An example remote data block ("Conference Event") has been registered.
+[Launch the plugin in WordPress Playground](https://playground.wordpress.net/?blueprint-url=https://raw.githubusercontent.com/Automattic/remote-data-blocks/trunk/blueprint.json) and explore. An example remote data block ("Conference Event") has been registered.
```

Comment:
I wonder if this should take the user into a post page with content being displayed, rather than the RDB config screen?

Reply:
> I wonder if this should take the user into a post page with content being displayed, rather than the RDB config screen?

@pkevan @maxschmeling It already does that via [this PR](https://github.com/Automattic/remote-data-blocks/pull/562). Can that content be improved? It's a simple wxr file hosted in `example/assets/`

Reply:
Probably worth merging that PR if it improves it, then iterate?